code/modules/surgery/organs/organ_internal.dm

```diff
@@ -76,6 +76,8 @@
 	return ..()
 
 /obj/item/organ/internal/proc/prepare_eat()
+	if(robotic >= 2)
```

Comment:
assisted is robotic 1..so they might just peel the assited parts off like an organge